mappings/DocumentMapping.json

```diff
@@ -0,0 +1,415 @@
+            },
+            "category": {
+              "type": "object",
+              "properties": {
+                "id": {
+                  "type": "keyword"
+                },
+                "name": {
+                  "type": "keyword"
+                }
+              }
+            }
+          }
+        },
+        "secondary_classification": {
+          "type": "nested",
+          "properties": {
+            "group": {
+              "type": "object",
+              "properties": {
+                "id": {
+                  "type": "keyword"
+                },
+                "name": {
+                  "type": "keyword"
+                }
+              }
+            },
+            "archive": {
+              "type": "object",
+              "properties": {
+                "id": {
+                  "type": "keyword"
+                },
+                "name": {
+                  "type": "keyword"
+                }
+              }
+            },
+            "category": {
+              "type": "object",
+              "properties": {
+                "id": {
+                  "type": "keyword"
+                },
+                "name": {
+                  "type": "keyword"
+                }
+              }
+            }
+          }
+        },
+        "proxy": {
+          "type": "keyword"
+        },
+        "source": {
+          "properties": {
+            "flags": {
+              "type": "keyword"
+            },
+            "format": {
+              "type": "keyword"
+            },
+            "size_bytes": {
+              "type": "integer"
+            }
+          }
+        },
+        "submitter": {
+          "type": "object",
+          "properties": {
+            "email": {
+              "type": "keyword"
+            },
+            "name": {
+              "type": "text",
+              "analyzer": "folding"
+            },
+            "submitter_id": {
+              "type": "integer"
+            },
+            "is_author": {
+              "type": "boolean"
+            },
+            "author_id": {
+              "type": "keyword"
+            },
+            "orcid": {
+              "type": "keyword"
+            }
+          }
+        },
+        "title": {
+          "type": "text",
+          "analyzer": "simple",
+          "search_analyzer": "standard",
+          "search_quote_analyzer": "simple",
+          "fields": {
+            "english": {
+              "type": "text",
+              "analyzer": "english",
+              "search_analyzer": "english",
+              "search_quote_analyzer": "simple"
+            },
+            "tex": {
+              "type": "text",
+              "analyzer": "tex_analyzer",
+              "search_analyzer": "tex_analyzer",
+              "search_quote_analyzer": "simple"
+            },
+            "keyword": {
+              "type": "keyword"
+            }
+          }
+        },
+        "title_utf8": {
```

Comment:
Why is there title and title_utf8?

Reply:
`title` is the raw article title; `title_utf8` is the version where single-character texisms (e.g. \`a) are converted to their corresponding UTF-8 charactars (e.g. à).